docs/develop/Repositories.md

```diff
@@ -1,45 +1,84 @@
+
+The ElectionGuard Python library is the _reference implementation_ of the ElectionGuard SDK. It covers the entire suite of functionality and processes necessary to implement an end-to-end verifiable election as part a voting system:
+
+- Key generation ceremony
+- Ballot encryption
+- Tally ceremony
+- Ballot decryption
+
+It is designed to be _portable_ over _performant_, _universal_ over _Pythonic_ (although we do try to adhere to the [Zen of Python][zen-of-python]).
+
+Any real-world voting use case will need to incorporate the capabilities of the Python library to run an [end-to-end verifiable election][election-guard-verifiability] or [post-election audit][election-guard-post-election-audit].
+
+[:fontawesome-brands-github: Source][election-guard-python-source] | 
+[:fontawesome-brands-python: Package][election-guard-python-package] | 
+[:fontawesome-regular-file: Documentation][election-guard-python-documentation]
+
+!!! info "Community-sourced Java port"
+    Check out our [community contributions for a java port](../contribute/index.md).
+
+## <div style="display: flex; align-items: center;">ElectionGuard C++<span style="display: flex; align-items: center; margin-left: 16px"> ![C++][cpp-logo]![C][c-logo]![C#][c-sharp-logo] <span></div>
+
+The ElectionGuard C++ library performs ballot encryption. It is designed for devices that handle the user experience of voting. These devices follow the economics and performance of embedded systems: special-purpose devices designed for low cost. We assume Intel Atom class processor-level performance and Raspberry Pi 3 types of operating systems. In addition to the C++ library itself, there is some focus on target different standalone packages. This leads to the usage of other languages like C and C#.
+
+[:fontawesome-brands-github: Source][election-guard-cpp-source]
+
+
+## <div style="display: flex; align-items: center;">ElectionGuard Python API<span style="display: flex; align-items: center; margin-left: 16px"> ![Python][python-logo] <span></div>
+
+This API uses the `electionguard-python` package in a python API utilizing FastAPI. This exposes the full featured python ElectionGuard and allows users to create docker containers of the API. 
+
+[:fontawesome-brands-github: Source][election-guard-api-source]
+
+## <div style="display: flex; align-items: center;">ElectionGuard UI<span style="display: flex; align-items: center; margin-left: 16px;"> ![Typescript][typescript-logo] <span></div>
+
+This UI application features several typescript packages as a monorepo to reduce repeated tooling. It contains two web applications. One is an application to administer an election. The other is an application to display the election results. There are two helper packages, one functions as an api client and the other is a component library. 
```

Comment:
Capitalization: `TypeScript`

Also, I had to look up the term monorepo, so maybe in parenthesis add "(a version-controlled code repository that holds many projects)" unless I'm woefully behind on technology terminology (a possibility I wouldn't rule out)